creator-node/src/snapbackSM.js

```diff
@@ -88,21 +258,36 @@ class SnapbackSM {
 
     // Enqueue first state machine operation
     // TODO: Remove this line permanently prior to final check-in
-    // this.stateMachineQueue.add({ startTime: Date.now() })
+    this.stateMachineQueue.add({ startTime: Date.now() })
```

Comment:
Remove this line permanently prior to final check-in and a determination of cron schedule